api/Document.json

```diff
@@ -1759,7 +1759,7 @@
               "version_added": "6"
             },
             "ie": {
-              "version_added": false
+              "version_added": "9"
```

Comment:
Is this simply because `addEventListener` in the test didn't work before IE9?

Reply:
I tried this with `attachEvent` as well but had no luck there.  I have a hunch that there's something different about `attachEvent` that I'm not aware of, but if that's the case, I recall we talked about treating it as a lack of support.

Reply:
We discussed it at some length in https://github.com/mdn/browser-compat-data/pull/12907, but that ultimately landed on IE9. Let's just go with that, live a little and risk being wrong about decade old browser release :)